application/common/util-native/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+import tools.forma.android.config.NdkAbi
+import tools.forma.android.config.CMake
+
+androidNative(
+    packageName = "com.stepango.blockme.common.util.native",
+    owner = Teams.core,
+    abi = setOf(NdkAbi.ARM7, NdkAbi.ARM8, NdkAbi.X86),
+    buildSystem = CMake(
+        path = File(projectDir, "src/main/jni/CMakeLists.txt"),
+        version = "3.10.2"
```

Comment:
It looks like a global setting, should we move it to FormaConfiguration?

Reply:
Actually, i've been thinking about it... We could move version into global config, but we have to specify CMakeLists for each modules, that the one of many reasons why i didn't want to merge this PR.